libs/vscode/webview/src/lib/get-task-execution-schema.ts

```diff
@@ -94,24 +93,24 @@ export async function getTaskExecutionSchema(
             return;
           }
 
-          const builderOptions = await readBuilderSchema(
-            cliTaskProvider.getWorkspacePath(),
-            selection.architectDef.builder
```

Comment:
Gets the defaults: Stepping out to `verifyBuilderDefinition` similar to the other executors here where it reads the workspace and is including the target config options in the call to readBuilderSchema.